### ISSUE TITLE
The aria-activedescendant attribute is sometimes missing

### ISSUE THREAD

**Issue by t-hamano** [pacocoursey/cmdk#373]
I'd like to use a test site to illustrate the problem. After cloning and setting up this repository, you should be able to access http://localhost:3000/ by running `pnpm testsite`.

The `aria-activedescendant` attribute does not seem to be applied correctly in the following two scenarios:

### Initial render

When you run the test site, the item with the text "Item" is selected. However, there is no `aria-activedescendant` attribute. This attribute is applied correctly when you select an item with the keyboard or hover over it with the mouse.

https://github.com/user-attachments/assets/3de1f9a1-9cdb-43de-9eb3-09f9ec0dd193

### After filtering items

Search for the keyword "xxx", where "xxx" is the value of an item with the text ”Value”. The list should be filtered and the "Value" should be selected, but there is no `aria-activedescendant` attribute.

https://github.com/user-attachments/assets/3b2f1f8c-4ecc-42a9-b9b8-098a552eb263
